File: yxb144325004祁陈斌/Notes/Notes/Note.m
//
//  Note.m
//  Notes
//
//  Created by xsdlr on 14/11/17.
//  Copyright (c) 2014年 xsdlr. All rights reserved.
//

#import "Note.h"

@implementation Note

@dynamic message;
@dynamic time;
@dynamic type;

+ (NSString *)TEXT_TYPE {
    return @"text";
}
+(NSString *)IMAGE_TYPE {
    return @"image";
}
+(NSString *)DRAW_TYPE {
    return @"draw";
}

@end

File: yxb144325165潘云童/第四次作业/MyNote/MyNote/Note.swift
//
//  Note.swift
//  MyNote
//
//  Created by Joker on 14/11/22.
//  Copyright (c) 2014年 Joker. All rights reserved.
//

import Foundation
import CoreData

class Note: NSManagedObject {

    @NSManaged var image: String
    @NSManaged var text: String
    @NSManaged var time: String

}
